src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -327,6 +399,25 @@ public abstract class Debugger
         /// </summary>
         internal event EventHandler<EventArgs> NestedDebuggingCancelledEvent;
 
+        #region Runspace Debug Processing Events
+
+        /// <summary>
+        /// Event raised when a runspace debugger needs breakpoint processing
+        /// </summary>
+        public event EventHandler<ProcessRunspaceDebugEventArgs> ProcessRunspaceDebug;
```

Comment:
This is a bit confusing.  I'll change "ProcessRunspaceDebugEnd" to "RunspaceDebugProcessingCompleted" since that is much better.  

The other two should probably be "StartRunspaceDebugProcessing" and "CancelRunspaceDebugProcessing".

The reason these two events exist is to give a host the chance to debug these remote sessions directly , rather than let PowerShell provide a default debug experience.  For example VSCode may want to show multiple windows to debug multiple remote sessions simultaneously instead of using PowerShell default processing which debugs each session serially.